lib/vrl/stdlib/benches/benches.rs

```diff
@@ -605,6 +608,73 @@ bench_function! {
     }
 }
 
+bench_function! {
+    parse_apache_log_common => vrl_stdlib::ParseApacheLog;
+
+    literal {
+        args: func_args![value: r#"127.0.0.1 bob frank [10/Oct/2000:13:55:36 -0700] "GET /apache_pb.gif HTTP/1.0" 200 2326"#,
+                         format: "common"
+        ],
+        want: Ok(value!({
+            "host": "127.0.0.1",
+            "identity": "bob",
+            "user": "frank",
+            "timestamp": (DateTime::parse_from_rfc3339("2000-10-10T20:55:36Z").unwrap().with_timezone(&Utc)),
+            "message": "GET /apache_pb.gif HTTP/1.0",
+            "method": "GET",
+            "path": "/apache_pb.gif",
+            "protocol": "HTTP/1.0",
+            "status": 200,
+            "size": 2326,
+        })),
+    }
+}
+
+bench_function! {
+    parse_apache_log_combined => vrl_stdlib::ParseApacheLog;
```

Comment:
Thanks!

Just a note that it is preferable to combine the cases for `parse_apache_log` into one `bench_function!` similar to https://github.com/timberio/vector/blob/d19426daab45958101e4d3917b796f284cb21e66/lib/vrl/stdlib/benches/benches.rs#L115-L131

This results in better grouping in the output.

Reply:
Ah cool. Sorry, wasn't thinking!

Reply:
No worries! This file could probably benefit from some docs.